src/services/fetchers/MetalPriceApiFetcher.ts

```diff
@@ -51,7 +51,7 @@ export default class MetalPriceApiFetcher implements FeedFetcherInterface {
         throw new Error(response.data);
       }
 
-      const rate = response.data.rates[currency];
+      const rate = response.data?.rates[symbol];
```

Comment:
why this change?

Reply:
It was wrong, probably during the refactoring I changed it wrongly. Now, it works fine.